state-chain/pallets/cf-threshold-signature/src/migrations/migrate_signature_to_include_signer.rs

```diff
@@ -25,7 +25,9 @@ mod old {
 
 impl<T: crate::Config<I>, I: 'static> OnRuntimeUpgrade for Migration<T, I> {
 	fn on_runtime_upgrade() -> frame_support::weights::Weight {
-		let current_aggkey = Keys::<T, I>::get(CurrentKeyEpoch::<T, I>::get().unwrap()).unwrap();
+		let current_aggkey =
+			Keys::<T, I>::get(CurrentKeyEpoch::<T, I>::get().unwrap_or(Default::default()))
+				.unwrap_or(Default::default());
```

Comment:
Why do we default here (and elsewhere in this commit)? What are the cases when the key doesn't exist?